pyrsh.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import paramiko
+import argparse
+import sys
+from contextlib import contextmanager
+	
+def arguments():
+	# Create description scrit
+	parser = argparse.ArgumentParser(description='Remote command execution via ssh')
+	#  specify the name and description argyments
+	parser.add_argument('host', help='Hostname', type=str)
+	parser.add_argument('user', help='Your user', type=str)
+	parser.add_argument('password', help='Your password', type=str)
+	parser.add_argument('command', help='Your command', type=str)
+	parser.add_argument('port', nargs='?', default=22, help='Enter your port, default port=22', type=int)
+	result = parser.parse_args()
+	return result
+	
+@contextmanager
+def ssh(args):
+	try:
+		connect = paramiko.SSHClient()
+		# If you can not find the key when connecting to the server, then by default the key is "discouraged" and called SSHException.
+		connect.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+		connect.connect(hostname=args.host, port=args.port, username=args.user, password=args.password)
+	except paramiko.ssh_exception.AuthenticationException:
+		sys.stderr.write("Authentication failed.\n")
+		sys.exit(-1)
```

Comment:
@GitSergeyGit decision to exit the program should not be done here. It is client code responsibility